standard_schema/prerelease/HED8.3.0.mediawiki

```diff
@@ -487,13 +487,16 @@ Each term in this vocabulary has a human-readable description and may include ad
 **** Female <nowiki>[Biological sex of an individual with female sexual organs such ova.]</nowiki>
 **** Male <nowiki>[Biological sex of an individual with male sexual organs producing sperm.]</nowiki>
 **** Intersex <nowiki>[Having genitalia and/or secondary sexual characteristics of indeterminate sex.]</nowiki>
+**** Other-sex <nowiki>[A non-specific designation of sexual traits.]</nowiki>
 *** Ethnicity <nowiki>[Belong to a social group that has a common national or cultural tradition. Use with Label to avoid extension.]</nowiki>

```

Comment:
Shouldnt this refer to a population or culture within which the ethnicity references?

Reply:
Could you make a specific wording suggestion.  Ethnicity isn't in BIDS so we are not constrained by a particular definitions.   Thx @smakeig